template/tsconfig/base/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.web.json",
+  "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
+  "compilerOptions": {
+    "baseUrl": ".",
```

Comment:
`baseUrl` here is redundant because TypeScript >= 4.1 supports `paths` without `baseUrl`: https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#paths-without-baseurl

But if it presents, `paths` aliases will be resolved relative to `baseUrl`. So, to prevent users from accidentally adding an incorrect `baseUrl`, we add a dummy one here.